go.mod

```diff
@@ -248,3 +248,16 @@ replace (
 	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
 	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
 )
+
+// Replacements to resolve the rapid reset CVE. This should be removed
+// when the upstream patched versions are released and downstreamed.
+// https://access.redhat.com/security/cve/CVE-2023-44487
+// https://access.redhat.com/security/cve/CVE-2023-39325
```

Comment:
Can we make downstream github issues/Jira issues to make sure we come back to this?